<robot name="exp_big_obj">
  <link name="big_obj">
    <inertial>
      <origin xyz="0.0675 0.0675 0.0725"/>
      <mass value="0.08" />
      <inertia  ixx="0.0003645" ixy="0.0"  ixz="0.0"  iyy="0.0003645"  iyz="0.0"  izz="0.000729" />
    </inertial>
    <visual>
      <origin xyz="0.0675 0.0675 0.0725"/>
      <geometry>
        <cylinder length="0.145" radius="0.0675"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.0675 0.0675 0.0725"/>
      <geometry>
        <cylinder length="0.145" radius="0.0675"/>
      </geometry>
    </collision>
  </link>

  <gazebo reference="big_obj">
    <material>Gazebo/Green</material>
     <mu1>0.75</mu1>
     <mu2>0.75</mu2>
	 <kp>15000</kp>
  </gazebo>
</robot>
